README.md

```diff
@@ -2,9 +2,11 @@
 
 ## Development environment
 
-This project uses [VCCW](http://vccw.cc/). 
+This project uses [Docker](https://www.docker.com/) for development.
 
-To get up and running install [Vagrant](http://vagrantup.com) and [Virtualbox](http://virtualbox.org), then run `vagrant up` in the plugin's root directory.
+To get up and running install [Docker](https://www.docker.com/get-started), then run `docker-compose up -d` in the plugin's root directory.
+
+For the inital Wordpress setup run `chmod +x docker-provision.sh` and `./docker-provision.sh`.
```

Comment:
Could we add the proper permissions through Git?

Reply:
Yeah, good call!